packages/elogind.rb

```diff
@@ -33,13 +20,32 @@ class Elogind < Package
   depends_on 'glib'
 
   def self.build
-    system "meson --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX} -Dcgroup-controller=elogind \
--Dman=false -Ddefault-hierarchy=legacy -Ddefault-kill-user-processes=false -Dhalt-path=/sbin/halt \
--Drootlibexecdir=#{CREW_PREFIX}/libexec/elogind -Dreboot-path=/sbin/reboot _build"
-    system 'ninja -v -C _build'
+    system "meson #{CREW_MESON_LTO_OPTIONS} \
+            --strip \
+            -Dcgroup-controller=elogind \
+            -Dman=false \
+            -Ddefault-hierarchy=legacy \
+            -Ddefault-kill-user-processes=false \
+            -Dhalt-path=/sbin/halt \
+            -Drootlibexecdir=#{CREW_PREFIX}/libexec/elogind \
+            -Dreboot-path=/sbin/reboot \
+            _build"
+    Dir.chdir '_build' do
+      system 'meson', 'compile'
+    end
   end
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} ninja -C _build install"
+    system "mv #{CREW_DEST_DIR}/bin/* #{CREW_DEST_DIR}/usr/local/bin/"
+    system "mv #{CREW_DEST_DIR}/etc #{CREW_DEST_DIR}/usr/local/etc"
+    system "mv #{CREW_DEST_DIR}/lib/* #{CREW_DEST_DIR}/usr/local/lib/"
+    system "mv #{CREW_DEST_DIR}/lib64/* #{CREW_DEST_DIR}/usr/local/lib64" # Please help rubyize
```

Comment:
Rubyize?
```
FileUtils.mv 'bin', CREW_DEST_PREFIX
FileUtils.mv 'etc', CREW_DEST_PREFIX
FileUtils.mv ARCH_LIB, CREW_DEST_PREFIX
```
Test first.

Reply:
unfortunately that doesn't work because there is already a folder called #{CREW_DEST_PREFIX}/usr/local/bin, so i have to move all the contents of bin to usr/local/bin. that raised `File exists - /usr/local/tmp/crew/dest/usr/local/bin (Errno::EEXIST)`

Reply:
```
FileUtils.mv Dir.glob('bin/*'), "#{CREW_DEST_PREFIX}/bin"
FileUtils.mv Dir.glob('etc/*'), "#{CREW_DEST_PREFIX}/etc"
FileUtils.mv Dir.glob("#{ARCH_LIB}/*"), CREW_DEST_LIB_PREFIX
```